dev/devicelab/bin/run.dart

```diff
@@ -245,6 +266,10 @@ final ArgParser _argParser = ArgParser()
       }
     },
   )
+  ..addOption(
+    'abresultfile',
```

Comment:
nit: other options in our tools use the dash to separate words, e.g. `ab-result-file`.

Reply:
Done